selfdrive/thermald/thermald.py

```diff
@@ -121,6 +121,30 @@ def handle_fan_uno(max_cpu_temp, bat_temp, fan_speed, ignition):
 
   return new_speed
 
+prev_time = None
+error_integral = 0
+def handle_fan_tici(max_cpu_temp, bat_temp, fan_speed, ignition):
+  global prev_time, error_integral, prev_cpu_temp_error
+
+  CPU_TEMP_SETPOINT = 80 if ignition else 60
+  FAN_PWR_RANGE = (0, 80 if ignition else 30)
+  KI = 1e-5
+
+  if prev_time is None:
+    prev_time = time.monotonic()
+  dt = max(time.monotonic() - prev_time, 0.01)
```

Comment:
does this work? looks like prev_time is only set once

Reply:
lol, that's why it gets unstable after long drives, good catch!